src/main/minio.js

```diff
@@ -403,7 +403,6 @@ export class Client {
           this.logHTTP(reqOptions, null, e)
           cb(e)
         })
-      req.once('finish', () => pipe.removeAllListeners('error'));
```

Comment:
what about 
- using a once for the error
- removing the listener after using the callback everywhere (can also prevent using the callback twice)
- using the end event (may not work/be necessary)